src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -49,22 +49,6 @@
  */
 public abstract class AbstractJavadocCheck extends AbstractCheck {
 
-    /**
-     * Message key of error message. Missed close HTML tag breaks structure
-     * of parse tree, so parser stops parsing and generates such error
-     * message. This case is special because parser prints error like
-     * {@code "no viable alternative at input 'b \n *\n'"} and it is not
-     * clear that error is about missed close HTML tag.
-     */
-    public static final String MSG_JAVADOC_MISSED_HTML_CLOSE =
-            JavadocDetailNodeParser.MSG_JAVADOC_MISSED_HTML_CLOSE;
-
-    /**
-     * Message key of error message.
-     */
-    public static final String MSG_JAVADOC_WRONG_SINGLETON_TAG =
-            JavadocDetailNodeParser.MSG_JAVADOC_WRONG_SINGLETON_TAG;
-
```

Comment:
Don't we need to remove the actual messages? Also please check our documentation for mentions of these.